diff --git a/tools/make_distrib.py b/tools/make_distrib.py
index 5e76b37..d749920 100644
--- a/tools/make_distrib.py
+++ b/tools/make_distrib.py
@@ -687,14 +687,7 @@ cef_paths2 = cef_paths2['variables']
 
 # Determine the build directory suffix. CEF uses a consistent directory naming
 # scheme for GN via GetAllPlatformConfigs in gn_args.py.
-if options.x64build:
-  build_dir_suffix = '_GN_x64'
-elif options.armbuild:
-  build_dir_suffix = '_GN_arm'
-elif options.arm64build:
-  build_dir_suffix = '_GN_arm64'
-else:
-  build_dir_suffix = '_GN_x86'
+build_dir_suffix = '_GN'
 
 # Determine the build directory paths.
 out_dir = os.path.join(src_dir, 'out')
