From 4a58309a95db9f5d17f8bc858efd80043bd0a034 Mon Sep 17 00:00:00 2001
From: nmlynch94 <nmlynch94@gmail.com>
Date: Wed, 8 Oct 2025 22:08:22 -0400
Subject: [PATCH] cef remove widevine assertions

---
 BUILD.gn         | 3 ---
 tools/gn_args.py | 2 --
 2 files changed, 5 deletions(-)

diff --git a/BUILD.gn b/BUILD.gn
index cc4af4032..830569ad6 100644
--- a/BUILD.gn
+++ b/BUILD.gn
@@ -255,9 +255,6 @@ if (is_linux) {
 # Enable support for Print Preview.
 assert(enable_print_preview)
 
-# Enable support for Widevine CDM.
-assert(enable_widevine)
-
 if (is_mac || is_win) {
   # Enable Widevine CDM host verification and storage ID.
   assert(enable_cdm_host_verification)
diff --git a/tools/gn_args.py b/tools/gn_args.py
index 023a90153..b2d9220b6 100644
--- a/tools/gn_args.py
+++ b/tools/gn_args.py
@@ -278,8 +278,6 @@ def GetRequiredArgs():
   """
   result = {
       'optimize_webui': True,
-      # Enable support for Widevine CDM.
-      'enable_widevine': True,
 
       # Don't use the chrome style plugin.
       'clang_use_chrome_plugins': False,
-- 
2.51.0

